src/components/HorizontalMenu/components/SubMenu/SubMenu.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useCallback } from 'react';
+
+import classNames from 'classnames/bind';
+import { useLocation } from 'react-router-dom';
+
+import { Menu, MenuItem } from '@components/Menu';
+
+import { TabProps } from '../../horizontal-menu-types';
+import { getMenuId } from '../../utils';
+import styles from './SubMenu.module.css';
+
+export type MenuProps = {
+  closeMenu: VoidFunction;
+  hoveredTabId: number | null;
+  menuAnchorEl: HTMLElement | null;
+  menuItemsWithSubMenu: Array<TabProps>;
+  selectTabAndNavigate: (tabId: number, path: string) => unknown;
+};
+
+const cn = classNames.bind(styles);
+
+export function SubMenu({
+  closeMenu,
+  hoveredTabId,
+  menuAnchorEl,
+  menuItemsWithSubMenu,
+  selectTabAndNavigate,
+}: MenuProps) {
+  const location = useLocation();
+
+  const isMenuItemSelected = useCallback(
+    (pathName: string): boolean => {
+      return location.pathname === pathName;
+    },
+    [location.pathname],
+  );
+
+  return (
+    <>
+      {menuItemsWithSubMenu.map(
+        (tab) =>
+          tab.subMenuItems!.length > 0 && (
```

Comment:
it could be realized with optional chaining or you should check an existing of this property

Reply:
i fixed this in the next PR https://github.com/EvgeniyKvasyuk/MAHE_WP/pull/21/files#diff-1e947ebf5514cd578523099be1d94d8cf9a23c77f8d2b8077fc66c79420db110